### PR TITLE
bump Sentry Android to 7.6.0 and Wizard to 3.21.0

### PR DESCRIPTION
**Pull request by lucas-zimerman**
This PR will bump Sentry Android to version 7.6.0 and Wizard to 3.21.0.
No break changes for the end user has been found.

NIT: fix on sample, crash wasn't defined but the SDK defines nativeCrash so I just renamed it.

Part of #319

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unrelease
```
Comment (lucas-zimerman) [inline CHANGELOG.md:3]:
```suggestion
## Unreleased
```

File: sample/www/js/events.js [hunk 2]
```diff
@@ -42,12 +42,12 @@ function addBreadcrumb(){ {
...
-      });    
+      });
 }}
 
 function setTag(){ {
     console.log("execute settag");
     var Sentry = cordova.require("sentry-cordova.Sentry");
     Sentry.setTag("tag", "value");
 
-}}
```
Comment (krystofwoldrich) [inline sample/www/js/events.js:53]: L: Nit, missing a new line at the end of the file.

**Review by krystofwoldrich — APPROVED** [review]
LGTM 🚀 Thank you

File: sample/www/js/events.js [hunk 3]
```diff
@@ -42,12 +42,12 @@ function addBreadcrumb(){ {
...
\ No newline at end of file
+}}
```
Comment (lucas-zimerman) [inline sample/www/js/events.js:53]:
```suggestion
}}

```

**Pull request merged**